pyhms/tree.py

```diff
@@ -309,29 +309,34 @@ def plot_best_fitness(self, filepath: str | None = None) -> None:
             fig.write_image(filepath)
         fig.show()
 
-    def plot_deme_variance(
+    def plot_deme_indicator(
         self,
         filepath: str | None = None,
         deme_id: str | None = "root",
         selected_dimensions: list[int] | None = None,
+        indicator: str | None = "AvgVar",
```

Comment:
Plotting a metric by its name system is a bit unclear from the user perspective, but on the other hand, it has it's merits. I like the ease of implementing new indicators. Maybe metric is a better name, than indicator?